hack/generate_changelog.sh

```diff
@@ -20,30 +20,26 @@ this=`basename $0`
 usage () {
 cat << EOF
 Generate a changelog for the specified tag
-Usage: $0 --for-tag <starting_tag> [--remote <remote_name>]
+Usage: $0 --reference <starting_tag> [--remote <remote_name>]
```

Comment:
Nit: It's not a "starting_tag" anymore.

Also, it's not clear if this generates the log from a specific tag.